src/library_strings.js

```diff
@@ -101,7 +101,7 @@ mergeInto(LibraryManager.library, {
    * @param {number} ptr
    * @param {number=} maxBytesToRead - An optional length that specifies the
    *   maximum number of bytes to read. You can omit this parameter to scan the
-   *   string until the first \0 byte. If maxBytesToRead is passed, and the string
+   *   string until the first \\0 byte. If maxBytesToRead is passed, and the string
```

Comment:
How about `null` or `NULL` instead?

Reply:
Yeah that works for me too, just wasn't sure if you'd be open to that. Or, as @kripken says, just "0 byte" also sounds good, I'll probably go with that.